GitHubContents.ps1

```diff
@@ -358,7 +362,12 @@ filter Set-GitHubContent
 
     $uriFragment = "/repos/$OwnerName/$RepositoryName/contents/$Path"
 
-    $encodedContent = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($Content))
+    if ($ContentPath) {
+        $encodedContent = [Convert]::ToBase64String((Get-Content -path $ContentPath -AsByteStream -Raw))
```

Comment:
We should also have a new [test](https://github.com/microsoft/PowerShellForGitHub/blob/master/Tests/GitHubContents.tests.ps1) that validates this new functionality as well.

Reply:
Added a test